Update CLI reference for 2.17.0 (#432)
okteto/docs: src/content/reference/cli.mdx

#### create

> Create a namespace. By default the command will switch to the new namespace, add `--use=false` for just creating the namespace and keeping the current one active.

Reviewer: ```suggestion
Create a namespace. By default the command will switch to the new namespace, add `--use=false` to create the namespace, and it will keep the current namespace active.
```

Non-blocking suggestion based on what I think I understand of this copy 😄
